modules/nf-scil/tracking/localtracking/main.nf

```diff
@@ -0,0 +1,118 @@
+process TRACKING_LOCALTRACKING {
+    tag "$meta.id"
+    label 'process_single'
+
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'scil.usherbrooke.ca/containers/scilus_1.6.0.sif':
+        'biocontainers/scilus/scilus:1.6.0' }"
+
+    input:
+    tuple val(meta), path(wm), path(fodf), path(fa)
+
+    output:
+    tuple val(meta), path("*__local_tracking.trk"), emit: trk
+    tuple val(meta), path("*__local_tracking_config.json"), emit: config
+    tuple val(meta), path("*__local_seeding_mask.nii.gz"), emit: seedmask
+    tuple val(meta), path("*__local_tracking_mask.nii.gz"), emit: trackmask
+    path "versions.yml"           , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def prefix = task.ext.prefix ?: "${meta.id}"
+
+    def local_fa_tracking_mask_threshold = task.ext.local_fa_tracking_mask_threshold ? task.ext.local_fa_tracking_mask_threshold : ""
+    def local_fa_seeding_mask_threshold = task.ext.local_fa_seeding_mask_threshold ? task.ext.local_fa_seeding_mask_threshold : ""
+    def local_tracking_mask = task.ext.local_tracking_mask_type ? "${task.ext.local_tracking_mask_type}" : ""
+    def local_seeding_mask = task.ext.local_seeding_mask_type ? "${task.ext.local_seeding_mask_type}" : ""
+
+    def local_step = task.ext.local_step ? "--step" + task.ext.local_step : ""
```

Comment:
Add a space after --step
```suggestion
    def local_step = task.ext.local_step ? "--step " + task.ext.local_step : ""
```
Same thing for --seed, --min_length and --max_length

Reply:
done